packages/vuetify/src/components/VInput/VInput.ts

```diff
@@ -272,26 +272,22 @@ export default baseMixins.extend<options>().extend({
       }, slot)
     },
     genPrependSlot () {
-      const slot = []
+      const slot = getSlot(this, 'prepend') || []
```

Comment:
**getSlot** can return `undefined`. I would assign it to a variable and if it's present, then push it, similar to what it's doing now.